tests/e2e/adminPage.spec.ts

```diff
@@ -0,0 +1,159 @@
+// adminPage.spec.ts
+import {
+  test,
+  expect,
+  chromium,
+  Browser,
+  BrowserContext,
+  Page
+} from "@playwright/test"
+import { AdminPage } from "./page_objects/adminPage"
+
+test.describe.serial("Admin Page", () => {
+  let browser: Browser
+  let context: BrowserContext
+  let page: Page
+
+  test.beforeAll(async () => {
+    // Create a new browser instance
+    browser = await chromium.launch()
+    context = await browser.newContext()
+    page = await context.newPage()
+
+    // Emulator admin account or Dev admin account
+    const adminEmail =
+      process.env.TEST_ADMIN_USERNAME ?? "testadmin@example.com"
+    const adminPassword = process.env.TEST_ADMIN_PASSWORD ?? "password"
```

Comment:
I recommend just letting the test fail if `process.env.TEST_ADMIN_USERNAME` or `process.env.TEST_ADMIN_PASSWORD` don't exist. We shouldn't have usernames or passwords hardcoded anywhere since it's a security concern.

Reply:
You won't have these credentials locally though, so we should figure out some way for devs to be able to get these credentials when they're writing tests  @Mephistic

Reply:
👍 We should be able to have the local credentials just rely on a user in the test fixture database (which is never deployed and already checked in anyway) - looks like we've updated to put that local user's credentials in `.env` to support local development.